packages/alea-frontend/pages/api/job-portal/check-org-invitations.ts

```diff
@@ -0,0 +1,27 @@
+import { NextApiRequest, NextApiResponse } from 'next';
+import { checkIfGetOrSetError, executeDontEndSet500OnError } from '../comment-utils';
+
+export async function checkInviteToOrg(
```

Comment:
Any function that calls `res.status` must have something like `set500OnError` in the name